WcaOnRails/app/webpacker/components/SearchWidget/IncidentItem.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function IncidentItem({
+  item,
+}) {
+  return (
+    <div className="omnisearch-item-incident">
+      Incident:
```

Comment:
i18n?

Reply:
Ah right (the title of incidents can't be localized though)!

This gave me the opportunity to test the "new" way locales are automatically reloaded and it simply works great :)